itests/transfer_with_sponsorship_api_test.go

```diff
@@ -0,0 +1,379 @@
+package itests
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	f "github.com/wavesplatform/gowaves/itests/fixtures"
+	"github.com/wavesplatform/gowaves/itests/testdata"
+	utl "github.com/wavesplatform/gowaves/itests/utilities"
+	"github.com/wavesplatform/gowaves/itests/utilities/issue_utilities"
+	"github.com/wavesplatform/gowaves/itests/utilities/sponsor_utilities"
+	"github.com/wavesplatform/gowaves/itests/utilities/transfer_utilities"
+	"github.com/wavesplatform/gowaves/pkg/crypto"
+)
+
+type TransferWithSponsorshipApiTxSuite struct {
+	f.BaseSuite
+}
+
+func (suite *TransferWithSponsorshipApiTxSuite) TestTransferWithSponsorshipApiPositive() {
+	waitForTx := true
+	versions := transfer_utilities.GetVersions()
+	for _, v := range versions {
+		//предусловия
+		//Аккаунт Sponsor выпускает токен
```

Comment:
It's nice that you've written comprehensive comments, but use English for them, please